hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientNearCacheTestSupport.java

```diff
@@ -629,6 +631,46 @@ public void run() {
         });
     }
 
+    protected void testNearCacheMemoryCostCalculation(InMemoryFormat inMemoryFormat, int threadCount) {
```

Comment:
Since `OBJECT` format does not perform memory cost calculation, why not return immediately in this case?